content/en-us/tutorials/fundamentals/coding-4/repeating-code-with-while-loops.md

```diff
@@ -121,12 +121,12 @@ With the while loop created, instructions can be added to change the part's colo
 
 ### Making the Script Wait
 
-If you add a second line of color changing code right now, it would change the brick's color so fast you might not even see the first color go by. To make the script wait before running the next line of code, use a `Library.task.wait()` function.
+If you add a second line of color changing code right now, it would change the brick's color so fast you might not even see the first color go by. This also causes the experience to crash. To make the script wait before running the next line of code, use a `Library.task.wait()` function.
```

Comment:
```suggestion
If you add a second line of color changing code right now, it would change the brick's color so fast you might not even see the first color go by. To make the script wait before running the next line of code, use a `Library.task.wait()` function.
```

Reply:
Hey @Flikter, even if this is true today, hopefully it won't always be true, and we're already advising readers not to do it. 😄